vgscheckout/src/androidTest/java/com/verygoodsecurity/vgscheckout/util/AccessTokenHelper.kt

```diff
@@ -0,0 +1,45 @@
+package com.verygoodsecurity.vgscheckout.util
+
+import com.google.gson.JsonParser
+import com.verygoodsecurity.vgscheckout.BuildConfig
+import okhttp3.*
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class AccessTokenHelper {
+
+    companion object {
+        private const val ACCESS_TOKEN = "access_token"
+        private fun parseToken(body: String?) = body?.let {
```

Comment:
@Endorf If this function is used only in this class, we can move it to the class body.